web/src/routes/MyPages/SurveyResPage/index.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
Use full name on every components.

It might confuse others. Since this page could be
- Survey Response Page
- Survey Result Page